src/LocalMixedGraphSpectralTargets.hpp

```diff
@@ -117,6 +120,8 @@ class UpscaleParameters
                        "--no-coarse-components", "Store trace, bubble components of coarse M.");
         args.AddOption(&coarse_factor, "--coarse-factor", "--coarse-factor",
                        "Coarsening factor for metis agglomeration.");
+        args.AddOption(&rescale_iter, "--rescale-iter", "--rescale-iter",
```

Comment:
What does this actually do? Is the hybridization solver more effective if this number is larger (at the cost of higher setup cost?)

Reply:
This number determines the way we compute the "rescaling vector" for the hybridized system. Like you said, we want a vector in the "near nullspace". Currently there are two ways to find it in the code. The first way is to approximate it by applying a few smoothing steps to `Ax = 0` with a nonzero initial guess (this was in the code before this PR). The second way to set it as some special vector based on Prop. 4.3 in SISC, 41(3), B425–B447 (newly added to this PR). 

When `rescale_iter` is negative, the rescaling vector is computed using the second way. This is now default because it seems to work pretty well, and it is very cheap to obtain (matrix-vector multiplication).

When `rescale_iter` is positive, the rescaling vector is computed using the first way, and `rescale_iter` indicates the number of smoothing steps.

When `rescale_iter` is 0, the hybridized system will not be rescaled.


To answer your second question, yes in general it is true that the solver is more effective when `rescale_iter` is larger.